games-simulation/simutrans_pak128/simutrans_pak128-2.6.recipe

```diff
@@ -0,0 +1,41 @@
+SUMMARY="Pakset for Simutrans"
+DESCRIPTION="A 'pakset' is the data set that Simutrans needs to run: \
+it contains all the data for the graphics of individual items (vehicles, \
+roads, railways, buildings, etc.) and all of the balancing information \
+(such as prices, speeds, capacities and so forth) necessary to make \
+the game complete.
+When Simutrans could only support 64px size graphics, pak128 already \
+started. First pak to feature a complex economy and have a very wide \
+variety of objects. It contains roughly 7 time more graphics data than \
+pak64 and thus requires by far the largest amount of RAM and \
+processing power of all Simutrans sets."
+HOMEPAGE="http://www.simutrans.com"
+SOURCE_URI="https://downloads.sourceforge.net/project/simutrans/pak128/pak128%20for%20ST%20120%20(2.6%2C%20completed%20elevated%20tracks)/pak128-2.6--ST120.zip"
+CHECKSUM_SHA256="ac0835ff667f80a2990ae226e8a58a9c44bbd1c2bf4a9f0f3cd87614ef9da94f"
+SOURCE_FILENAME="simupak_128-$portVersion.zip"
+SOURCE_DIR=""
+COPYRIGHT="2004-2007 Tomas Kubes
+	2008-2016 The pak128-Team"
+LICENSE="Artistic"
+REVISION="1"
+ARCHITECTURES="any"
+
+PROVIDES="
+	simutrans_pak128 = $portVersion
+	simutrans_pakset = 120.2 compat >= 120.2
+	"
+
+GLOBAL_WRITABLE_FILES="
+	non-packaged/$relativeAppsDir/simutrans/pak128/config directory keep-old
+	"
+
+INSTALL()
+{
+	mkdir -p $appsDir/simutrans
+	mkdir -p $prefix/non-packaged/$relativeAppsDir/simutrans/pak128/config
+	cp -r pak128 $appsDir/simutrans/
+	mv $appsDir/simutrans/pak128/config \
+		$prefix/non-packaged/$relativeAppsDir/simutrans/pak128/
```

Comment:
Moving things to non-packaged essentially leaves them out of the package. Are you sure this works? Or does it just happen to work because you have simutrans datafiles copied manually to your non-packaged directory?

The package should put things either in $appsDir/simutrans somewhere, or use $dataDir/simutrans/.